packages/http/src/validator/utils/spec.ts

```diff
@@ -5,8 +5,22 @@ export function findOperationResponse(
   statusCode: number,
 ): IHttpOperationResponse | undefined {
   const sortedSpecs = responseSpecs
-    .filter(spec => new RegExp(`^${spec.code.replace(/X/g, '\\d')}$`).test(String(statusCode)))
-    .sort((s1, s2) => s1.code.split('X').length - s2.code.split('X').length);
+    .filter(
+      spec =>
+        new RegExp(`^${spec.code.replace(/X/g, '\\d')}$`).test(String(statusCode)) ||
+        spec.code.toLowerCase() === 'default',
+    )
+    .sort((s1, s2) => {
+      if (s1.code.toLowerCase() === 'default') {
+        return 1;
```

Comment:
Question: shouldn't this be -1? Ideally we want default to be the last choice, not the first one.

Reply:
`1` on s1 moves it down.

<img width="323" alt="Screen Shot 2019-09-25 at 14 23 57" src="https://user-images.githubusercontent.com/7136002/65600358-4496b680-dfa0-11e9-8e31-dbfc667000b8.png">